### PR TITLE
Teradata Plugins

### PR DESCRIPTION
**Pull request by amolkokane-cirus**
Added missing javadocs and fixed pom.xml versions
Changed copyright to 2020

### CODE REVIEW

File: teradata-plugin/pom.xml [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+  <artifactId>teradata-plugin</artifactId>
+  <name>Teradata plugin</name>
+  <version>1.4.0-SNAPSHOT</version>
+  <modelVersion>4.0.0</modelVersion>
+
+  <dependencies>
+    <dependency>
+      <groupId>io.cdap.cdap</groupId>
+      <artifactId>cdap-etl-api</artifactId>
```
Comment (rmstar) [inline teradata-plugin/pom.xml:35]: the scope should be provided.
https://github.com/data-integrations/google-cloud/blob/develop/pom.xml#L150
Please take a look at above pom file for scope of the artifacts

**Review by rmstar — APPROVED** [review]

File: teradata-plugin/pom.xml (hunk 1 above)
Comment (amolkokane-cirus) [reply teradata-plugin/pom.xml:35]: This dependency has been already declared with scope as "provided" in the parent pom.xml file. Do we still have to explicitly mention the scope here? We checked pom.xml files from other sub-projects under database-plugins and none of them has scope explicitly mentioned. Can you please confirm?

Comment (rmstar) [reply teradata-plugin/pom.xml:35]: Oh I see. Please ignore my comment

**Comment by shiva-cirus** [comment]
Squash complete.

**Pull request merged**